### PR TITLE
Delete Getting Started page.

### LINKED ISSUE #59: [BUG] Unnecessary 'get started' screen
**Describe the bug**
The 'get started' screen after the user agreement seems to be unnecessary, since the next screen provides sufficient instructions to the user.
image

### LINKED ISSUE #76: [BUG] "Click here" doesn't work on a phone
## Context ##
"You've joined the CovidNearMe community" invites people to "Click here to get started," even though this is a mobile product with no mouse.

## Suggestions ##
- Delete this page entirely (#59)
- Change the text to something more actionable like "Record how you're feeling today", or at least
- Use the verb "tap" instead of "click"

## Screenshot ##
https://photos.google.com/share/AF1QipPNzbQ9YfdnsdI-nSG2WbrB9J5y35_mu5uOORQ4X2pNHiUvaFTdWdgB3X9Y6WxfBw/photo/AF1QipN_a5B9peHEDNyT7HDV9qvncandHySuOrmb5EC5?key=am52QmZoVTZqOWxxWURVVE9EdkJpYUM3LWxfdGlB

## Smartphone: ##
 - Device: Essential
 - OS: Android 10 QP1A.191005.014
 - App version: 6d80da8

### PR DESCRIPTION
**Pull request by gspencergoog**
This removes the unnecessary interstitial screen after consent is obtained, and combines it with the home screen. The welcome message is shown there until a first checkup is completed, and then it is hidden.

I also updated action buttons to be CAPS, per the Material spec, and changed some of the button text to be clearer actions.

Also fixes the localization files: they were broken at HEAD.

Fixes #59 
Fixes #76 

New home screen looks like this after the consent screen:
image

### CODE REVIEW

File: lib/src/l10n/app_localizations_en.dart [hunk 1]
```diff
@@ -248,6 +245,5 @@ class AppLocalizationsEn extends AppLocalizations {
       'You seem to be offline. Please check your network settings and try again.';
 
   @override
-  String get deniedConsentBackButton =>
-      'Go back to the informed consent screen';
+  String get deniedConsentBackButton => 'GO BACK';
```
Comment (clocksmith) [inline lib/src/l10n/app_localizations_en.dart]: This label is for a tooltip on the back icon button. Is it better to leave as is for a more descriptive semantics node?

File: lib/src/l10n/app_en.arb [hunk 2]
```diff
@@ -331,7 +325,7 @@
      "description":"Message shown on the banner displayed when no network connection is detected"
   },
 
-  "deniedConsentBackButton": "Go back to the informed consent screen",
+  "deniedConsentBackButton": "GO BACK",
```
Comment (clocksmith) [inline lib/src/l10n/app_en.arb]: This label is for a tooltip on the back icon button. Is it better to leave as is for a more descriptive semantics node?

**Review by clocksmith — DISMISSED** [review]

File: lib/src/l10n/app_en.arb (hunk 2 above)
Comment (gspencergoog) [reply lib/src/l10n/app_en.arb]: Ahh, it used to be a regular button... Reverted.

**Comment by gspencergoog** [comment]
Sorry @clocksmith, I force-pushed, and I guess that invalidates your approval. Can you do it again, please?

**Review by digiter — APPROVED** [review]
LGTM

**Pull request merged**